cves/CVE-2018-13304.yml

```diff
@@ -365,7 +380,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 617
```

Comment:
I've never encountered `Reachable Assertion` before. A quick note here to explain would be cool, especially if this is marketed for newer security folks who may only know the "main" ones like buffer overflows and the like.

Nice job overall!